tools/ci_build/github/azure-pipelines/templates/py-packaging-selectable-stage.yml

```diff
@@ -395,6 +395,10 @@ stages:
         EpBuildFlags: --use_tensorrt --use_tensorrt_builtin_parser --tensorrt_home="C:\local\TensorRT-8.6.0.12.Windows10.x86_64.cuda-11.8" --cuda_version=$(CUDA_VERSION) --cuda_home="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v$(CUDA_VERSION)" --cmake_extra_defines "CMAKE_CUDA_ARCHITECTURES=37;50;52;60;61;70;75;80"
         EnvSetupScript: setup_env_gpu.bat
         EP_NAME: gpu
+        ${{if contains(Agent.Name, 'onnxruntime-gpu-winbuild-t4')}}:
```

Comment:
All of the Windows GPU jobs in this pipeline and "python packaging pipeline" should use "onnxruntime-gpu-tensorrt8-winbuild-t4" machine pool until the TRT team has a new pool with VS2022.